### ISSUE TITLE
Accessibility: ToolStrip: TabStop= true, pressing Shift-Tab for first or last item: toolStripComboBox/TextBox in ToolStrips doesn't move to right item right 

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#5795]
* .NET Core Version: 
.Net SDK 6.0.100-rc.2.21466.17

* Have you experienced this same bug with .NET Framework?: 
Yes, but the result is different from .net 6.0 after the PR: https://github.com/dotnet/winforms/pull/5726

**Minimal repro:**
Scenario A:
1.	Create a Winforms .Net Core application with 2 toolStrips, set their TabStop as true
2.	In the first toolStrip, the last item is not toolStripComboBox/TextBox, and in the second toolStrip, the last item is toolStripComboBox/TextBox
3.	Build and run app
4.	Press Shift-Tab several times
[WinFormsApp14.zip](https://github.com/dotnet/winforms/files/7183233/WinFormsApp14.zip)

Scenario B:
1.	Create a Winforms .Net Core application with 2 toolStrips, set their TabStop as true
2.	In the first toolStrip, the first  item is not toolStripComboBox/TextBox, and in the second toolStrip, the first item is toolStripComboBox/TextBox
3.	Build and run app
4.	Press tab firstly to the first item in the second toolStrip, then press Shift-Tab 
[WinFormsApp15.zip](https://github.com/dotnet/winforms/files/7183239/WinFormsApp15.zip)

**Actual behavior:**
For Scenario A: trap in the last item: toolStripComboBox/TextBox of the second toolStrip
![ShiftTabIssue1](https://user-images.githubusercontent.com/26474449/133738689-412380ae-6179-4d65-8e2b-4ab86a489382.gif)
For Scenario B: it is not in the last item for the first toolStrip, it is in the last item of current toolStrip after pressing Shift-Tab
![ShiftTabIssue2](https://user-images.githubusercontent.com/26474449/133738749-def6a376-f3d7-435e-a5bb-f7d87021a19f.gif)

**Expected behavior:**
For Scenario A: not trap in the last item: toolStripComboBox/TextBox of the second toolStrip
For Scenario B: it is in the last item of the first toolStrip after pressing Shift-Tab

**Comment by ArtemTatarinov** [comment]
Hi @Olina-Zhang, @merriemcgaw, @Tanya-Solyanik, could you please tell me if I'm getting this right: pressing `Shift`+`Tab` with `TabStop`=`true` should always move the focus to the first `ToolStripItem` of a previous `ToolStrip`?

For example, we could stand on any item of the second `ToolStrip`, but if we press `Shift`+`Tab`, we should move the focus to the first item of the first `ToolStrip`? And if we stand on any item of the first `ToolStrip`, we should move cyclically to the first item of the second `ToolStrip` when `Shift`+`Tab` is pressed?

**Comment by merriemcgaw** [comment]
@ArtemTatarinov that is not the preferred behavior. 

What we need to do is if the user does `Shift`+`Tab` we should simply go to the previous item in the current `ToolStrip`  where `TabStop`=`true`. If we are at the first item in a `ToolStrip` the `Shift`+`Tab` should take the user to the last item in the previous `ToolStrip` that has `TabStop`=`true`. It basically the opposite of what the `Tab` alone should do.

**Comment by merriemcgaw** [comment]
I think I must have misunderstood the first explanation @ArtemTatarinov, what you describe above is precisely how I'd like it to work. What is the timeline you need to get it into that state? I'd love to try to get this into the 6.0 release - that would be our last bug from the test pass and we'd be without any MAS violations left!
